eng/scripts/Language-Settings.ps1

```diff
@@ -14,7 +14,12 @@ function Get-java-PackageInfoFromRepo ($pkgPath, $serviceDirectory)
     $projectData.load($projectPath)
     $projectPkgName = $projectData.project.artifactId
     $pkgVersion = $projectData.project.version
-    $pkgGroup = $projectData.project.groupId
+    if ($projectData.project.psobject.properties.name -contains "groupId") {
```

Comment:
```suggestion
    if ($projectData.project -contains "groupId") {
```

Reply:
That code doesn't work. the contains operator needs to be on a list of the property names.

Reply:
You are right. I was a little confused by the introduction of `psobject`. But the code is fine the way it was.